modules/sql-engine/src/integrationTest/sql/set/test_intersect.test

```diff
@@ -74,7 +74,7 @@ statement ok
 CREATE TABLE t_float(f1 float);
 
 statement ok
-INSERT INTO t_float(f1) VALUES (-1.2345678901234e+200), (-34.84),  (123), (456), (456.1), (-456);
+INSERT INTO t_float(f1) VALUES (-1.2345678901234e+10), (-34.84),  (123), (456), (456.1), (-456);
```

Comment:
This value can not be represented as a float value but constant literals didn't overflow prior to this  patch.